### PR TITLE
I set right-bar default to file list in a 1-1 direct chat room

### PR DESCRIPTION
**Pull request by jacobvm04**
Fixed https://github.com/vector-im/riot-web/issues/8719
Signed-off-by: Jacob Vanmeter <jacobvm04@gmail.com>
This is my first time contributing to this project, so please tell me if I did something incorrectly.

### CODE REVIEW

**Comment by Half-Shot** [comment]
Hmm, I wonder if there is an issue for what to do about the memberlist for 1:1s. File list seems reasonable to me.

File: src/components/structures/RightPanel.js [hunk 1]
```diff
@@ -190,7 +204,7 @@ export default class RightPanel extends React.Component {
 
         return (
             <aside className={classes}>
-                { panel }
+                {panel}
```
Comment (t3chguy) [inline src/components/structures/RightPanel.js]: I think this change is unintended as the code-style at least used to prefer spaced when children

File: src/components/views/rooms/RoomHeader.js [hunk 2]
```diff
@@ -201,8 +202,8 @@ module.exports = React.createClass({
         const emojiTextClasses = classNames('mx_RoomHeader_nametext', { mx_RoomHeader_settingsHint: settingsHint });
         const name =
             <div className="mx_RoomHeader_name" onClick={this.props.onSettingsClick}>
-                <EmojiText dir="auto" element="div" className={emojiTextClasses} title={roomName}>{ roomName }</EmojiText>
-                { searchStatus }
+                <EmojiText dir="auto" element="div" className={emojiTextClasses} title={roomName}>{roomName}</EmojiText>
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: Same as above

File: src/components/views/rooms/RoomHeader.js [hunk 3]
```diff
@@ -201,8 +202,8 @@ module.exports = React.createClass({
...
+                {searchStatus}
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: Same as above

File: src/components/views/rooms/RoomHeader.js [hunk 4]
```diff
@@ -213,7 +214,7 @@ module.exports = React.createClass({
             }
         }
         const topicElement =
-            <div className="mx_RoomHeader_topic" ref="topic" title={topic} dir="auto">{ topic }</div>;
+            <div className="mx_RoomHeader_topic" ref="topic" title={topic} dir="auto">{topic}</div>;
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: Same as above

File: src/components/views/rooms/RoomHeader.js [hunk 5]
```diff
@@ -244,18 +245,18 @@ module.exports = React.createClass({
 
             pinnedEventsButton =
                 <AccessibleButton className="mx_RoomHeader_button mx_RoomHeader_pinnedButton"
-                                  onClick={this.props.onPinnedClick} title={_t("Pinned Messages")}>
-                    { pinsIndicator }
+                    onClick={this.props.onPinnedClick} title={_t("Pinned Messages")}>
+                    {pinsIndicator}
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: Same as above

File: src/components/views/rooms/RoomHeader.js [hunk 6]
```diff
@@ -296,24 +297,29 @@ module.exports = React.createClass({
...
-                { shareRoomButton }
-                { manageIntegsButton }
-                { forgetButton }
-                { searchButton }
+                {settingsButton}
+                {pinnedEventsButton}
+                {shareRoomButton}
+                {manageIntegsButton}
+                {forgetButton}
+                {searchButton}
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: Same as above (for these 6 lines)

File: src/components/views/rooms/RoomHeader.js [hunk 7]
```diff
@@ -296,24 +297,29 @@ module.exports = React.createClass({
...
-                    { topicElement }
-                    { cancelButton }
-                    { rightRow }
-                    <RoomHeaderButtons collapsedRhs={this.props.collapsedRhs} />
+                    <div className="mx_RoomHeader_avatar">{roomAvatar}</div>
+                    {e2eIcon}
+                    {name}
+                    {topicElement}
+                    {cancelButton}
+                    {rightRow}
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: Same as above (for these 6 lines)

File: src/components/structures/RightPanel.js (hunk 1 above)
Comment (t3chguy) [reply src/components/structures/RightPanel.js]: This not being documented in the code-style but simply following the style of the rest of the code base (consistency is most important)

**Comment by unroasted** [comment]
that was my auto formatter, I will fix it soon.

**Comment by unroasted** [comment]
I fixed all of those styling issues, tell me if I did anything wrong.

**Comment by lampholder** [comment]
I'm afraid I am a little worried about this change.

Right now, a room's being a DM is only a flag - it can have two users, or more. The only way a user knows exactly who they're talking to is through the memberlist.

The most important psycological benefit of a dm is that it gives the user a sense of trust and privacy - that they're talking in a small, controlled group and that they're not going to be taken by surprise and accidentally share something with people they didn't mean to.

I worry that this change being made in isolation would reduce that sense of confidence, privacy and trust.

**Comment by jryans** [comment]
Thanks for making this contribution a while back.

It looks like there are some concerns that this isn't the right approach to take, so I think it's best to close this for now.

If you are still interested in pursuing this feature or something similar, please discuss with us in #riot-dev:matrix.org to find a good approach forward.

**Pull request closed without merging**